examples/07_advanced/plot_localizer_mass_univariate_methods.py

```diff
@@ -57,6 +57,9 @@
 fmri_masked = nifti_masker.fit_transform(contrast_map_filenames)
 
 
+""
+contrast_map_filenames
```

Comment:
WDYM ?

Reply:
Ah, I didn't mean to push this change. It's just an artifact from when I was debugging the example. I'll remove it.